client/flutter/lib/home_page.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:WHOFlutter/generated/l10n.dart';
+import 'package:WHOFlutter/page_button.dart';
+import 'package:WHOFlutter/page_scaffold.dart';
+import 'package:WHOFlutter/pages/protect_yourself.dart';
+import 'package:WHOFlutter/pages/travel_advice.dart';
+import 'package:WHOFlutter/pages/who_myth_busters.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:share/share.dart';
+
+class HomePage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return PageScaffold(
+        body: Column(
+      mainAxisSize: MainAxisSize.max,
+      mainAxisAlignment: MainAxisAlignment.center,
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: <Widget>[
+        Spacer(flex: 1),
+        PageButton(
+            title: S.of(context).protectYourself,
+            onPressed: () => Navigator.of(context)
+                .push(MaterialPageRoute(builder: (c) => ProtectYourself()))),
+        PageButton(
+            title: S.of(context).whoMythBusters,
+            onPressed: () => Navigator.of(context)
+                .push(MaterialPageRoute(builder: (c) => WhoMythBusters()))),
+        PageButton(
+            title: S.of(context).travelAdvice,
+            onPressed: () => Navigator.of(context)
+                .push(MaterialPageRoute(builder: (c) => TravelAdvice()))),
+        PageButton(
+            title: S.of(context).shareTheApp,
+            lightColor: true,
+            onPressed: () => Share.share(
+                'Check out the official COVID-19 GUIDE App https://preview.whoapp.org/menu')),
```

Comment:
Not blocking check-in this has to go away

Reply:
hm.  easily changed if you have a suggestion for tonight...

Reply:
That URL would be a url to the app itself on the appropriate app store.

We have no idea what the bundle id is for Apple yet for example.